design/design.md

```diff
@@ -0,0 +1,112 @@
+Applications?
+mithril, peras, leios
+
+What cryptographic primitives will we implement?
+I propose that we implement the generic ALBA primitive to make it as applicable
```

Comment:
@tolikzinovyev : maybe add a reference to what you mean by "generic ALBA" - it's maybe not clear to everyone.